src/mono/CMakeLists.txt

```diff
@@ -180,12 +182,16 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
   set(HOST_DARWIN 1)
   set(PTHREAD_POINTER_ID 1)
   set(USE_MACH_SEMA 1)
+  if (GC_SUSPEND STREQUAL "default")
+	set(GC_SUSPEND "hybrid")
+  endif()
 elseif(CMAKE_SYSTEM_NAME STREQUAL "iOS")
   # See man cmake-toolchains(7) on which variables
   # control cross-compiling to ios
   add_definitions(-DNO_GLOBALIZATION_SHIM)
   add_definitions(-D_THREAD_SAFE)
   set(HOST_DARWIN 1)
+  set(HOST_IOS 1)
```

Comment:
Isn't iOS the target, not the host? If we rely on `HOST_IOS` defines I think we should clean those up. We admittedly have a bunch of various incorrect host vs target defines, but ideally we use this as a chance to clean them up rather than leave in hacks. I'll probably root out a bunch as part of the multi-target cross compiler work as well.

Reply:
Host is the platform the code will run on, so this should be correct.

Reply:
Host is the platform the binaries will run on, target is the platform targeted by the JIT/AOT compiler. So for the ios cross compiler, OSX will be the host, and IOS will be the target.

Reply:
Right, which is why I'm confused why `HOST_IOS` is a valid define here.

Reply:
Its inside a
elseif(CMAKE_SYSTEM_NAME STREQUAL "iOS")
block.